nautobot/docs/plugins/development.md

```diff
@@ -146,6 +147,7 @@ The configurable attributes for a `PluginConfig` are listed below in alphabetica
 | `description` | Brief description of the plugin's purpose |
 | `graphql_types` | The dotted path to the list of GraphQL type classes (default: `graphql.types.graphql_types)` |
 | `installed_apps` | A list of additional Django application dependencies to automatically enable when the plugin is activated (you must still make sure these underlying dependent libraries are installed) |
+| `jinja_filters` | The dotted path to the file that contains jinja filters to be registered (default: `jinja_filters.jinja_filters`) |
```

Comment:
Remove? I don't see where any code is using this setting - it reads to me that jinja filters are being automatically discovered just by being included in a `jinja_filters.py`, with no need to explicitly list them in a separate `jinja_filters` list variable.